packages/@aws-cdk/aws-ecs/lib/container-definition.ts

```diff
@@ -562,6 +575,51 @@ function renderUlimit(ulimit: Ulimit): CfnTaskDefinition.UlimitProperty {
     hardLimit: ulimit.hardLimit,
   };
 }
+/**
+ * Container dependency
```

Comment:
Can you please add the link to ContainerDependency in the documentation: https://docs.aws.amazon.com/AmazonECS/latest/APIReference/API_ContainerDependency.html

Reply:
Added that link and brought the documentation in line with the v1.0 style I saw elsewhere in the file.